.github/workflows/eunit.yaml

```diff
@@ -22,10 +22,16 @@ jobs:
         - debian10
         - debian9
         - centos8
-        - centos7
+        #- centos7
 
-    container: emqx/build-env:${{ matrix.erl_otp }}-${{ matrix.os }}
+    container: "ghcr.io/emqx/emqx-builder/emqx/emqx-builder:${{ matrix.erl_otp }}-${{ matrix.os }}"
 
     steps:
     - uses: actions/checkout@v2
+    - name: Install flex on ubuntu20.04
+      if: matrix.os == 'ubuntu20.04'
+      run: apt-get update && apt-get install -y flex bison
+    - name: Install flex on debian10
+      if: matrix.os == 'debian10'
+      run: apt-get update && apt-get install -y flex bison
```

Comment:
there should be no need for these.

Reply:
sorry, they are indeed messing in the builder. we'll have to add them to the builder image